decidim-core/app/packs/stylesheets/decidim/map.scss

```diff
@@ -1,3 +1,38 @@
 @import "stylesheets/decidim/plugins/leaflet";
 @import "stylesheets/decidim/plugins/MarkerCluster";
 @import "stylesheets/decidim/plugins/MarkerCluster.Default";
+
+/* overwrite leaflet and card styles */
+.leaflet-popup-content-wrapper{
+  @apply bg-black w-fit rounded text-left text-white transition-all ease-in delay-300;
```

Comment:
@Crashillo what's the delay for? 

Right now you have no transition to show it (or it is not applied), and transition to hide it. I would put transition for both (remember to apply a transition to all hovered/toggled elements), and a little

Reply:
`leaflet-popup-content-wrapper` is a default Leaflet css class, hence I overrode some of those styles, like the transition properties (actually there is a fade out on close).

I presume the last part of the message has been chopped out :wink:

Reply:
Hummm, it turns out that the delay is in order to achieve a synchronization between the popup and the arrow

[Screencast from 20-02-23 11:53:38.webm](https://user-images.githubusercontent.com/817526/220085609-b4a4a173-9fb3-4032-86e1-a281e45f0729.webm)

Once both things are in sync, when the popup fades in, the transition is missing... :thinking:  still working on that.

Reply:
I figured out you cannot alter the popup transition duration, if you do, fade-in/fade-out timings are out of sync. So the current transition is the one by default